panic: protoc has error but buf does not: /Users/wroden/.cache/buf/Darwin/x86_64/bin/protoc returned error: exit status 1 buf/buf.proto:9:9: "buf.bar" is already defined in file "buf/bar/bar.proto".


goroutine 1 [running]:
github.com/bufbuild/buf/internal/buf/bufimage/bufimagebuild/bufimagebuildtesting.(*fuzzResult).panicOrN(0xc000293bd0, 0x193f220, 0xc00012c000, 0x1bb)
	/Users/wroden/repos/WillAbides/buf/internal/buf/bufimage/bufimagebuild/bufimagebuildtesting/bufimagebuildtesting.go:205 +0x16c
github.com/bufbuild/buf/internal/buf/bufimage/bufimagebuild/bufimagebuildtesting.Fuzz(0x3570000, 0x1bb, 0x1bb, 0x4)
	/Users/wroden/repos/WillAbides/buf/internal/buf/bufimage/bufimagebuild/bufimagebuildtesting/bufimagebuildtesting.go:49 +0x118
go-fuzz-dep.Main(0xc0004dff70, 0x1, 0x1)
	go-fuzz-dep/main.go:36 +0x1b8
main.main()
	github.com/bufbuild/buf/internal/buf/bufimage/bufimagebuild/bufimagebuildtesting/go.fuzz.main/main.go:15 +0x52
exit status 2